docs/deployment/version_skew_policy.md

```diff
@@ -0,0 +1,70 @@
+# Version Skew Policy
```

Comment:
Does it make sense to drop now the short section that was introduced with https://github.com/gardener/gardener/pull/4917 as this document provides much more details on the topic?

Reply:
Yep, I've dropped it. thanks!